repository.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ESPHome Home Assistant Add-Ons",
+  "name": "ESPHome",
   "url": "https://github.com/esphome/hassio",
```

Comment:
I think you should link this to the ESPHome project instead.

```suggestion
  "url": "https://esphome.io",
```

Reply:
Thanks